chore: dependencies and logging

marnixdessing opened the pull request.

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4867016767

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #234 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4877781417

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #235 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4888579209

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #238 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4898617746

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #239 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4904344718

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #240 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4910072180

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #241 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
chore: add additional logging to debug invalid_request

joostvanderborg commented on src/app/auth/handleRequest.ts:
```diff
@@ -15,7 +15,7 @@ export async function handleRequest(
   logsClient: CloudWatchLogsClient,
 ) {
-  console.debug('Creating session object...');
+  console.debug('Creating session object...', cookies, dynamoDBClient);
```
gezien je hiermee session hijacking wel erg gemakkelijk maakt: kunnen we de cookies enigszins anonimiseren (keys + eerste paar karakters waarde of zo?)

marnixdessing replied:
Hmm lastig. De cookie string is een semicolon separated string van cookies `<naam>=<value>;` format. Ik weet niet precies wat er fout gaat bij het maken van het session object. Maar er gaat iets dusdanig fout dat de hele runtime niet eens met een error faalt (de invalid_request dus). Wellicht kan ik eerst nog extra tests toevoegen aan het Session object.

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4920912798

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #243 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4931737012

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #244 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4942551620

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #245 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4953792827

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #246 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4964046526

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #247 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4969578670

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #248 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4975212471

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #249 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4986453776

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #250 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/4997937394

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #251 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
fix: bug in familyname surname fields

marnixdessing committed:
Merge branch 'acceptance' of github.com:GemeenteNijmegen/yivi-issue-app into acceptance

marnixdessing committed:
fix: tests

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

marnixdessing commented:
Deze fix kan niet omdat we de cookie string anders loggen.
